extensions/sql-migration/src/constants/strings.ts

```diff
@@ -253,6 +282,40 @@ export function AZURE_SQL_TARGET_PAGE_DESCRIPTION(targetInstance: string = 'inst
 	return localize('sql.migration.wizard.target.description', "Select an Azure account and your target {0}.", targetInstance);
 }
 
+export const AZURE_SQL_TARGET_CONNECTION_ERROR_TITLE = localize('sql.migration.wizard.connection.error.title', "An error occurred while conneting to the target server.");
+export function SQL_TARGET_CONNECTION_ERROR(message: string): string {
+	return localize('sql.migration.wizard.target.connection.error', "Connection error: {0}", message);
+}
+export function SQL_TARGET_CONNECTION_SUCCESS(databaseCount: string): string {
+	return localize('sql.migration.wizard.target.connection.success', "Connection was successful. Target databases found: '{0}'", databaseCount);
```

Comment:
Remove the quotes around the number of databases:
![image](https://user-images.githubusercontent.com/11844501/185511628-f4048dab-4d3c-4f33-96a1-bc283bbc4bfa.png)